### PR TITLE
[Merged by Bors] - chore(Order/Basic): cleanup redundant type arguments

### PR DESCRIPTION
**Pull request by madvorak**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Order/Basic.lean [hunk 1]
```diff
@@ -1503,7 +1502,7 @@ instance Prop.partialOrder : PartialOrder Prop where
 
 end «Prop»
 
-variable {s : β → β → Prop} {t : γ → γ → Prop}
+variable {γ : Type w} {s : β → β → Prop} {t : γ → γ → Prop}
```
Comment (YaelDillies) [inline Mathlib/Order/Basic.lean:1505]: Why not keep `γ` at the top?

File: Mathlib/Order/Basic.lean [hunk 2]
```diff
@@ -66,7 +65,7 @@ open Function
 
 universe u v w
 
-variable {ι : Type*} {α : Type u} {β : Type v} {γ : Type w} {π : ι → Type*} {r : α → α → Prop}
+variable {ι : Type*} {α : Type u} {β : Type v} {π : ι → Type*}
```
Comment (YaelDillies) [inline Mathlib/Order/Basic.lean:68]: While you're at it
```suggestion
variable {ι α β : Type*} {π : ι → Type*}
```

File: Mathlib/Order/Basic.lean (hunk 1 above)
Comment (madvorak) [reply Mathlib/Order/Basic.lean:1505]: It is used here for the first time. Why to clutter the infoview for the intermediate 1500 lines?

File: Mathlib/Order/Basic.lean (hunk 2 above)
Comment (madvorak) [reply Mathlib/Order/Basic.lean]: Do you want to get rid of `universe u v w` altogether?

File: Mathlib/Order/Basic.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/Order/Basic.lean:1505]: Type variables are usually put at the top

Comment (madvorak) [reply Mathlib/Order/Basic.lean:1505]: OK

Comment (madvorak) [reply Mathlib/Order/Basic.lean]: Actually, the entire line 1506 was never used, lol.

File: Mathlib/Order/Basic.lean (hunk 2 above)
Comment (madvorak) [reply Mathlib/Order/Basic.lean]: I deleted them.

File: Mathlib/Order/Basic.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/Order/Basic.lean]: Welp, that solves it

**Comment by YaelDillies** [comment]
maintainer merge

**Pull request closed without merging**